shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -270,6 +314,21 @@ public void detachAccessibiltyBridge() {
         accessibilityEventsDelegate.setAccessibilityBridge(null);
     }
 
+    public void attachTextInputPlugin(TextInputPlugin textInputPlugin) {
```

Comment:
Can you add javadocs for all public methods? And in those docs, for these attach/detach methods, can you describe when attach and detach should be called?

Reply:
done